src/util.c

```diff
@@ -159,12 +154,11 @@ void print_test_results(struct ntttcp_test_endpoint *tep)
 	if (tep->test->verbose) {
 		PRINT_INFO("\tThread\tTime(s)\tThroughput");
 		PRINT_INFO("\t======\t=======\t==========");
-		for (i=0; i<tep->total_threads; i++) {
+		for (i = 0; i < tep->total_threads; i++) {
 			if (tep->results->threads[i]->is_sync_thread == true)
 				continue;
 
-			log_tmp = format_throughput(tepr->threads[i]->total_bytes,
```

Comment:
can we keep them into 2 lines as current is

Reply:
done